ContentfulDeliveryAPI/CDAConfiguration.h

```diff
@@ -16,7 +16,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-extern const NSString* const CDA_DEFAULT_SERVER;
+extern NSString* const CDA_DEFAULT_SERVER;
```

Comment:
Yes, @mariobodemann, to confirm, it is NOT writable: in this case, the const is actually applied to the value. See this [SO answer](http://stackoverflow.com/a/3196554/4068264)